frontend/catalyst/api_extensions/quantum_operators.py

```diff
@@ -147,6 +147,7 @@ def circuit():
     if postselect is not None and postselect not in [0, 1]:
         raise TypeError(f"postselect must be '0' or '1', got {postselect}")
     in_classical_tracers.append(postselect)
+    in_classical_tracers.append(reset)
```

Comment:
Just want to leave a todo here for Mudit's other PR. I think since these values are not tracers they would be better stored in the `MidCircuitMeasure` class directly:
```suggestion
    # TODO: Move these values into separate attributes of the MidCircuitMeasure class
    in_classical_tracers.append(postselect)
    in_classical_tracers.append(reset)
```